operator/main.go

```diff
@@ -197,6 +205,14 @@ func defineFlagVar() *FlagVar {
 	flag.BoolVar(&flagVar.enableModuleCatalog, "enable-module-catalog", true,
 		"Enabling the Module Catalog Synchronization for Introspection of "+
 			"available Modules based on ModuleTemplates.")
+	flag.BoolVar(&flagVar.enableKcpWatcher, "enable-kcp-watcher", true,
+		"Enabling KCP Watcher to reconcile Watcher CRs created by KCP run operators")
+	flag.StringVar(&flagVar.skrWatcherPath, "skr-watcher-path", "./internal/skr/chart/skr-webhook",
```

Comment:
```suggestion
	flag.StringVar(&flagVar.skrWatcherPath, "skr-watcher-path", "skr-webhook",
```
This is not correct path after deployed with docker